cli/build.rs

```diff
@@ -1,28 +1,35 @@
 //! Build script to extract git hash of iroha build and to check runtime validator
 
-use eyre::{eyre, Result};
-use vergen::{vergen, Config};
+use std::error::Error;
 
 const DEFAULT_VALIDATOR_PATH: &str = "../default_validator";
 
-fn main() -> Result<()> {
+fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
1. The reason it was an `eyre::Result` was to support complex backtraces and `wrap_err`. Removal is fine. 
2. This is a main function, where nothing can be forwarded up the call stack. 

Just a plain `unwrap` in here gives you the same information. Either approach is fine, I just don't like `Box<dyn Error>` in my code base.

Reply:
To be honest, for the life of me I don't understand why I removed `eyre` in the first place. I just added it back in.